pod-operation/src/components/lim_temperature.rs

```diff
@@ -21,9 +34,10 @@ impl LimTemperature {
 		self.ads1015.destroy_ads1015();
 	}
 
-	pub fn read_pins(&mut self) -> (i16, i16, i16, i16) {
+	pub fn read_pins(&mut self) -> (f32, f32, f32, f32) {
```

Comment:
Chore: since this function now returns temperatures rather than raw ADC measurements, the name doesn't quite match its purpose.